packages/terra-functional-testing/src/terra-dev-site/terra-functional-testing/About.1.tool.mdx

```diff
@@ -1,4 +1,4 @@
-# WebDriverIO Testing
+# Terra Functional Testing
```

Comment:
could you add badges here?

Reply:
```suggestion
import { Badge } from '@cerner/terra-functional-testing/package.json?dev-site-package';

<Badge />

# Terra Functional Testing
```

Reply:
Added: (saw the suggestion too late)

https://github.com/cerner/terra-toolkit/pull/574/commits/deabf23e0f84fb6fdab1f7034993d0a67d5f2530